assets/animations.js

```diff
@@ -13,10 +13,18 @@ function onIntersection(elements, observer) {
   });
 }
 
-function initializeScrollAnimationTrigger(rootEl = document) {
+function initializeScrollAnimationTrigger(event, rootEl = document) {
   const animationTriggerElements = Array.from(rootEl.getElementsByClassName(SCROLL_ANIMATION_TRIGGER_CLASSNAME));
   if (animationTriggerElements.length === 0) return;
 
+  if (Shopify.designMode) {
+    animationTriggerElements.forEach((element) => {
+      if (event) {
+        element.classList.add('scroll-trigger--in-viewport');
```

Comment:
We can improve this class name, as it's applied to all elements that have the `scroll-trigger` class on them and not just the ones currently in your viewport. I think this name would be better suited for when we solve the [above-the-fold issue](https://github.com/Shopify/themes/issues/400). How about something like `scroll-trigger--design-mode` or `scroll-trigger--editor-mode`?